lib/Controller/ConfigController.php

```diff
@@ -96,6 +97,23 @@ public function addAccount(string $url, string $token) {
 		}
 	}
 
+	public function updateAccountFilters(int $id, array $projects = [], array $groups = []) {
+		try {
+			$result = $this->accountMapper->updateFilters($id, $projects, $groups) === 1;
```

Comment:
This could be made more readable with a proper exception or at least a boolean return value